### ISSUE TITLE
On logout redirect doesn't respect baseURL

### ISSUE THREAD

**Issue by mfeckie** [simplabs/ember-simple-auth#1373]
According to README https://github.com/simplabs/ember-simple-auth#configuration `baseURL` can be set via config.

In the initializer, however, it is never respected if a rootURL is present. https://github.com/simplabs/ember-simple-auth/blob/master/app/initializers/ember-simple-auth.js#L11

Should this be removed from README if it's not to be used?

**Comment by marcoow** [comment]
Yes, actually the whole _"Configuration"_ section should be dropped. A PR would be awesome!

**Comment by mfeckie** [comment]
OK, done that.

Any tip for having a different 'redirect' URL that `rootURL` ?  My initial thought is to just override the initializer, but that worries me for future updates.

**Comment by marcoow** [comment]
I think the base way to support that would be to introduce a new `baseURL` property on the `ApplicationRouteMixin` that defaults to `Configuration.baseURL` (similar to [`AuthenticatedRouteMixin`'s `authenticationRoute` property](https://github.com/simplabs/ember-simple-auth/blob/master/addon/mixins/authenticated-route-mixin.js#L53)).

Alternatively you can just override the [`ApplicationRouteMixin`'s `sessionInvalidated` method](https://github.com/simplabs/ember-simple-auth/blob/master/addon/mixins/application-route-mixin.js#L135).
